app/components/find/courses/international_students_component/view.rb

```diff
@@ -28,6 +43,14 @@ def visa_type
         def sponsorship_availability
           @sponsorship_availability ||= course.public_send("can_sponsor_#{visa_type}") ? :available : :not_available
         end
+
+        def course_has_relocation_entitlement?
+          SUBJECTS_WITH_RELOCATION_ENTITLEMENT.include?(course_subject_codes.first)
```

Comment:
I think there are some annoying bugs here: 

1. If the first subject is Physics which has relocation entitlement, and the second subject is Biology which doesn't that works fine, but when you change the first subject to another relocation entitlement course, such as Latin (I think) it shifts the second subject (Biology) which doesn't have relocation entitlement to the first position in the array which means the text does not generate 🤯 

2. If you have modern languages (French) as the first subject, which has relocation entitlement the content renders correctly, however if you then add Biology as the second subject (which does not have relocation entitlement) the content does not render. It looks like it pushes French to the bottom of the array, maybe because it's `type: "ModernLanguagesSubject"` 🤷🏼‍♂️ 

Might be worth adding a few tests for these?

Reply:
This is a different underlying issue I raised before in our chats about the subject ordering. I encountered it when working on generating titles. I won't tackle that here because it requires a little more investigation. i think i may have already carded it up but if not will do

Reply:
Is it OK for candidates to see that content when they may not be entitled to the financial help? I guess the content says "May"

Reply:
Technically they wouldn't (which in terms of the api is correct because we want to rely on the first subject). This only goes funny when Modern Language is paired with a secondary subject (where the secondary subject gets swapped into the first slot). I've (reluctantly) handled it in the component for now but we need to solve the root problem otherwise we'll keep applying bandages everywhere. I count 3 different places where we're more or less doing the same thing to handle the issue.